tslearn/docs/index.rst

```diff
@@ -80,3 +80,4 @@ From here, you can navigate to:
     reference
     auto_examples/index
     contributing
+    integration_other_software
```

Comment:
I would rather add that one between DTW and API Reference. Such that we have a structure with: "important tutorials", then "api ref", then gallery and finally details for advanced users / contributors, how to cite, etc.

By the way, maybe at some point we could have a proper "Tutorials" section, not sure if that would help or not. WDYT @GillesVandewiele @johannfaouzi ?

Reply:
Yes that would most definitely be helpful! I have this in mind for the updated README (on which I'll try to work soon).

Reply:
I will change the order!

Some packages have a user guide:
* scikit-learn: http://scikit-learn.org/stable/user_guide.html
* imbalanced-learn: https://imbalanced-learn.readthedocs.io/en/stable/user_guide.html
* pyts: https://pyts.readthedocs.io/en/stable/user_guide.html
If you look for some inspiration!

Reply:
Yes, this is the kind of things I have in mind, except that I do not think we would need a dedicated page that would just serve as a table of contents for the user guide (but I'm not sure if it is easy to trick Sphinx without a dedicated TOC page...)

Reply:
You don't have to create a dedicated TOC page, but you need to reference each page somewhere so that users can find them (unless there is only one page for all the tutorials). The dedicated TOC page is just for convenience!